docs/UsersGuide/source/Components.rst

```diff
@@ -0,0 +1,81 @@
+
+.. WARNING::
+   For GFS data, dates prior to 1 January 2018 may work but are not guaranteed. Public archives of model data can be accessed through the `National Centers for Environmental Information <https://www.ncdc.noaa.gov/data-access/model-data/model-datasets/global-forcast-system-gfs>`_ (NCEI) or through the `NOAA Operational Model Archive and Distribution System <https://nomads.ncep.noaa.gov/>`_ (NOMADS). Raw external model data may be pre-staged on disk by the user.
+
+
+Forecast Model
+==============
+
+The prognostic atmospheric model in the UFS SRW Application is the Finite-Volume Cubed-Sphere
+(:term:`FV3`) dynamical core configured with a Limited Area Model (:term:`LAM`) capability :cite:`BlackEtAl2020`. The dynamical core is the computational part of a model that solves the equations of fluid motion. A User’s Guide for the UFS :term:`Weather Model` is `here <https://ufs-weather-model.readthedocs.io/en/ufs-v2.0.0/>`__. 
+
+Supported model resolutions in this release include 3-, 13-, and 25-km predefined Contiguous U.S. (:term:`CONUS`) domains, each with 64 vertical levels. Preliminary tools for users to define their own domain are also available in the release with full, formal support of these tools to be provided in future releases. The Extended Schmidt Gnomonic (ESG) grid is used with the FV3-LAM, which features relatively uniform grid cells across the entirety of the domain. Additional information about the FV3 dynamical core can be found `here <https://noaa-emc.github.io/FV3_Dycore_ufs-v2.0.0/html/index.html>`__ and on the `NOAA Geophysical Fluid Dynamics Laboratory website <https://www.gfdl.noaa.gov/fv3/>`_.
+
+Interoperable atmospheric physics, along with the Noah Multi-parameterization (Noah MP) Land Surface Model options, are supported through the Common Community Physics Package (:term:`CCPP`), described `here <https://dtcenter.org/community-code/common-community-physics-package-ccpp>`__. Atmospheric physics are a set of numerical methods describing small-scale processes such as clouds, turbulence, radiation, and their interactions. There are two physics options supported for the release. The first is an experimental physics suite being tested for use in the future operational implementation of the Rapid Refresh Forecast System (RRFS) planned for 2023-2024, and the second is an updated version of the physics suite used in the operational Global Forecast System (GFS) v15. A scientific description of the CCPP parameterizations and suites can be found in the `CCPP Scientific Documentation <https://dtcenter.ucar.edu/GMTB/v5.0.0/sci_doc/index.html>`_, and CCPP technical aspects are described in the `CCPP Technical Documentation <https://ccpp-techdoc.readthedocs.io/en/v5.0.0/>`_. The model namelist has many settings beyond the physics options that can optimize various aspects of the model for use with each
```

Comment:
There are 4 supported physics options supported for this release. See this doc: https://docs.google.com/spreadsheets/d/1m8RYA485SyMe0BSLya0rUJZKABOQMXJf7PVzufgulIQ/edit?pli=1#gid=0 to get the details needed to update this paragraph.

Reply:
This document has the physics suites listed, and I could potentially copy the table into the docs so people have more info, but I'm wondering why someone would choose one of these versus another. Is there any documentation that describes the purpose/benefits/uses of each physics suite? For example, I think some are better for high-resolution forecasts and others for low-resolution, if I recall correctly. It's unclear to me where in the CCPP docs I'd find that kind of info (if anywhere). Should I ask Ligia?

Reply:
There are different physics suites being tested/tuned by different organizations. It is possible that some of these would be used for a multi-physics suite ensemble in the future too. I don't know if there is documentation on what suite to chose when. You can ask Ligia but I am guessing that will be hard to come by.